test/functional/feature_bip68_sequence.py

```diff
@@ -29,7 +29,10 @@
 class BIP68Test(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 2
-        self.extra_args = [[], ["-acceptnonstdtxn=0"]]
+        self.extra_args = [
+            ["-acceptnonstdtxn=1"],
+            ["-acceptnonstdtxn=0"],
```

Comment:
Suggest you just remove this. We don't usually specify config in tests where it's the default.

Reply:
I think it is easier to read due to symmetry